<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnJoystick">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbButtons">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Кнопки</string>
    </property>
    <widget class="QLineEdit" name="edA">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbA">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>20</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>A</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edB">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>20</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>20</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edX">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbY">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>20</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edY">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbLBumper">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LBumper</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edLBumper">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbRBumper">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RBumper</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edRBumper">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edView">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>220</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbMenu">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Menu</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edMenu">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbAxis">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>191</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Оси</string>
    </property>
    <widget class="QLineEdit" name="edLStickX">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbLStickX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LStickX</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edLStickY">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbLStickY">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LStickY</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbRStickX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RStickX</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edRStickX">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbRStickY">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RStickY</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edRStickY">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbLTrigger">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LTrigger</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edLTrigger">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbRTrigger">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RTrigger</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edRTrigger">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
